### PR TITLE
Eliom_client.Page_status.*: stop parameter; function while_active

### PR DESCRIPTION
**Pull request by jrochel**
This parameter facilitates the deinstallation of a handler.

### CODE REVIEW

File: src/lib/eliom_client.client.ml [hunk 1]
```diff
@@ -544,22 +544,29 @@ module Page_status = struct
   end
 
   let maybe_just_once ~once e = if once then React.E.once e else e
+  let stop_event ?(stop = React.E.never) e =
+    ignore @@ React.E.map (fun () -> React.E.stop e) stop
```
Comment (vouillon) [inline src/lib/eliom_client.client.ml]: You need to use `React.E.stop ~strong:true`.
See http://erratique.ch/software/react/doc/React.html#strongstop

File: src/lib/eliom_client.client.ml [hunk 2]
```diff
@@ -567,6 +567,12 @@ module Page_status = struct
   let oninactive ?(once = false) ?stop action =
     stop_event ?stop @@
       React.E.map action @@ maybe_just_once ~once @@ Events.inactive ()
+
+  let while_active ?now ?(stop = React.E.never) action =
+    let thread = ref Lwt.return_unit in
+    onactive ?now ~stop (fun () -> thread := action ());
+    oninactive ~stop (fun () -> Lwt.cancel !thread);
+    ignore @@ React.E.map (fun () -> Lwt.cancel !thread) stop
```
Comment (vouillon) [inline src/lib/eliom_client.client.ml]: Is it possible that you cancel the thread twice?

Comment (jrochel) [reply src/lib/eliom_client.client.ml]: Selon mes expériences résilier un fil d'exécution deux fois ne pose pas de problème.

File: src/lib/eliom_client.client.ml [hunk 3]
```diff
@@ -544,22 +544,29 @@ module Page_status = struct
   end
 
   let maybe_just_once ~once e = if once then React.E.once e else e
+  let stop_event ?(stop = React.E.never) e =
+    ignore @@ React.E.map (fun () -> React.E.stop ~strong:true e) stop
```
Comment (vouillon) [inline src/lib/eliom_client.client.ml]: Is there anything that prevents the `page_status` signal from being stopped as well?
See http://erratique.ch/software/react/doc/React.html#strongstop

Comment (jrochel) [reply src/lib/eliom_client.client.ml]: Now there is.

**Pull request merged**